src/lib/viewers/BaseViewer.js

```diff
@@ -1094,6 +1095,29 @@ class BaseViewer extends EventEmitter {
     handleAssetAndRepLoad() {
         this.loadBoxAnnotations().then(this.createAnnotator);
     }
+
+    /**
+     * Method to insert the viewer wrapper before any other children
+     *
+     * @param {Element} element Element to be inserted into the DOM
+     * @return {Element} inserted element
+     */
+    insertViewerWrapper(element) {
```

Comment:
Maybe generalize this to an insertBefore method that can live as util? You pass in the element and the container div that you want to stick it into.

Reply:
ok I'll create a util